ecs-cli/modules/cli/cluster/cluster_app_test.go

```diff
@@ -990,7 +991,7 @@ func TestClusterUpARM64(t *testing.T) {
 	flagSet := flag.NewFlagSet("ecs-cli-up", 0)
 	flagSet.Bool(flags.CapabilityIAMFlag, true, "")
 	flagSet.String(flags.KeypairNameFlag, "default", "")
-	flagSet.String(flags.ArchitectureFlag, "arm64", "")
+	flagSet.String(flags.InstanceTypeFlag, "a1.medium", "")
```

Comment:
could you also add error cases, maybe `a1foo.large`?

Reply:
The code I have right now can't catch that though- it just checks if the string starts with "a1". The only list of a1 instances that we have is in the CFN template.

Reply:
Oh I see your comment below about using a regex, nvm